crates/router/src/routes/payments.rs

```diff
@@ -2566,7 +2566,7 @@ pub async fn payments_finish_redirection(
 
     let locking_action = payload.get_locking_input(flow.clone());
 
-    api::server_wrap(
+    Box::pin(api::server_wrap(
```

Comment:
added this to address clippy v2 error
![Screenshot 2025-02-18 at 14 45 37](https://github.com/user-attachments/assets/a753763c-2502-4b3c-abe0-5436c0d64aae)